zim/notebook/notebook.py

```diff
@@ -352,7 +352,7 @@ def info(self):
 			uri = self.uri
 		except AssertionError:
 			uri = None
-
+		from . import NotebookInfo
```

Comment:
Please move this import to the top of the file

Reply:
Can't - it'd be a circular import and fail. Both `__init__` and `info` import stuff from `notebook`